.rubocop.yml

```diff
@@ -0,0 +1,188 @@
+  Enabled: true
+
+Layout/SpaceAfterSemicolon:
+  Description: "Use spaces after semicolons."
+  StyleGuide: '#spaces-operators'
+  Enabled: true
+
+Layout/SpaceBeforeComma:
+  Description: "No spaces before commas."
+  Enabled: true
+
+Layout/SpaceBeforeComment:
+  Description: >-
+                 Checks for missing space between code and a comment on the
+                 same line.
+  Enabled: true
+
+Layout/SpaceBeforeSemicolon:
+  Description: "No spaces before semicolons."
+  Enabled: true
+
+Layout/SpaceAroundOperators:
+  Description: "Use a single space around operators."
+  StyleGuide: "#spaces-operators"
+  Enabled: true
+
+Layout/Tab:
+  Description: "No hard tabs."
+  StyleGuide: "#spaces-indentation"
+  Enabled: true
+
+Layout/TrailingBlankLines:
+  Description: "Checks trailing blank lines and final newline."
+  StyleGuide: '#newline-eof'
+  Enabled: true
+
+Layout/TrailingWhitespace:
+  Description: "Avoid trailing whitespace."
+  StyleGuide: "#no-trailing-whitespace"
+  Enabled: true
+
+#################### Naming ##############################
+
+Naming/ClassAndModuleCamelCase:
+  Description: "Use CamelCase for classes and modules."
+  StyleGuide: "#camelcase-classes"
+  Enabled: true
+
+Naming/ConstantName:
+  Description: "Constants should use SCREAMING_SNAKE_CASE."
+  StyleGuide: "#screaming-snake-case"
+  Enabled: true
+
+#################### Style ###############################
+
+Style/ClassVars:
+  Description: 'Avoid the use of class variables.'
+  StyleGuide: '#no-class-vars'
+  Enabled: true
+
+Style/DefWithParentheses:
+  Description: "Use def with parentheses when there are arguments."
+  StyleGuide: "#method-parens"
+  Enabled: true
+
+Style/Encoding:
+  Description: "Use UTF-8 as the source file encoding."
+  StyleGuide: "#utf-8"
+  Enabled: true
+
+# TODO
+Style/FrozenStringLiteralComment:
+  Description: "Add frozen_string_literal comment to the top of files"
+  Include:
+  - "app/**/*.rb"
+  - "lib/**/*.rb"
+  Enabled: false
+
+Style/HashSyntax:
+  Description: "Prefer Ruby 1.9 hash syntax { a: 1, b: 2 } over 1.8 syntax"
+  EnforcedStyle: ruby19
+
+Style/IfInsideElse:
+  Description: "Finds if nodes inside else, which can be converted to elsif."
+  Enabled: true
+
+Style/MethodCallWithoutArgsParentheses:
+  Description: "Do not use parentheses for method calls with no arguments."
+  StyleGuide: "#method-invocation-parens"
+  Enabled: true
+
+Style/MethodDefParentheses:
+  Description: >-
+                 Checks if the method definitions have or don't have
+                 parentheses.
+  StyleGuide: "#method-parens"
+  Enabled: true
+
+Style/RedundantReturn:
+  Description: "Don't use return where it's not required."
+  Enabled: true
+
+Style/RedundantSelf:
+  Description: "Don't use self where it's not needed."
+  Enabled: true
+
+Style/TrailingCommaInArguments:
+  Description: "Checks for trailing comma in argument lists."
+  StyleGuide: "#no-trailing-params-comma"
+  Enabled: true
+
```

Comment:
Sorry, I don't understand your concern about this cop. What's the benefit of
```ruby
def foo(bar,)
  # do stuff
end
```
over
```ruby
def foo(bar)
  # do stuff
end
```
???

You can run `rubocop -a` and it will automatically remove these commas for you.

Reply:
Maybe I misunderstood. I thought it were about the kind of comma's at the end of this line https://github.com/rails-girls-summer-of-code/rgsoc-teams/pull/857/files#diff-ea406fc966c043b0342994808d837c01L65 
Of course I don't weep about the example you give above 😂